engine/src/main/java/org/terasology/rendering/nui/SortOrderSystem.java

```diff
@@ -262,7 +267,20 @@ public static void setUsed(ArrayList<Integer> other) {
         used = other;
     }
 
-    public static boolean getModifierPressed() {
-        return modifierPressed;
+    public static boolean containsConsole() {
+        if (enabledWidgets != null) {
+            boolean containsConsole = false;
+            for (CoreScreenLayer layer : enabledWidgets) {
+                if (layer instanceof ConsoleScreen) {
+                    LoggerFactory.getLogger("").info("console open");
+                    containsConsole = true;
```

Comment:
You can short circuit here by simply replacing this like so:
```suggestion
                    return true;
```
and then removing the next return below